app/assets/javascripts/controllers/vm_cloud/vm_cloud_resize_form_controller.js

```diff
@@ -0,0 +1,34 @@
+ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'miqService', function($http, $scope, vmCloudResizeFormId, miqService) {
+  $scope.vmCloudModel = {
+    flavor: null,
+  };
+  $scope.flavors = [];
+  $scope.formId = vmCloudResizeFormId;
+  $scope.modelCopy = angular.copy( $scope.vmCloudModel );
+
+  ManageIQ.angular.scope = $scope;
+
+  $http.get('/vm_cloud/resize_form_fields/' + vmCloudResizeFormId)
+    .then(getResizeFormData)
+    .catch(miqService.handleFailure);
+
+  $scope.cancelClicked = function() {
+    miqService.sparkleOn();
+    var url = '/vm_cloud/resize_vm/' + vmCloudResizeFormId + '?button=cancel';
+    miqService.miqAjaxButton(url);
+  };
+
+  $scope.submitClicked = function() {
+    miqService.sparkleOn();
+    var url = '/vm_cloud/resize_vm/' + vmCloudResizeFormId + '?button=submit';
+    miqService.miqAjaxButton(url, true);
```

Comment:
Please submit the angular model instead of relying on implicit form data serializization .. this should be `miqService.miqAjaxButton(url, vm.vmCloudModel)`